requirements-nvidia.txt

```diff
@@ -0,0 +1,5 @@
+cupy>=6
+dask-cuda
+jupyterlab==1.2.1
```

Comment:
The first two yes, these three I don't think are gpu/nvidia specific. May want to add it into docker requriements

Reply:
dask-cuda is from the RAPIDS.ai project and enables multi-gpu dask tasks.  
cupy is a requirement for dask-cuda.
jupyter-lab requires some changes in the start script.  This is not gpu specific.

Reply:
> cupy is a requirement for dask-cuda

so it can probably be dropped here? I'd be surprised if it were not listed in `dask-cuda`'s requirements already

> This is not gpu specific.

Strictly speaking, non of these are devito-GPU specific. These are the requirements to run DASK on a (multi-)GPU node/cluster. And DASK is used in one of the layers of abstraction on top of Devito.  So I think this file should be moved to a different location, or at least be renamed

and @mloubout , instead of three `requirements-*` files in the root dir, we should perhaps start thinking about a reorganization of the directory tree

Reply:
Yes we may need to reorganize a bit our requirements but for another PR I'd say